lite/kernels/imagination_nna/bridges/fc_op.cc

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "lite/core/subgraph_bridge_registry.h"
+#include "lite/kernels/imagination_nna/bridges/graph.h"
+
+namespace paddle {
+namespace lite {
+namespace subgraph {
+namespace imagination_nna {
+
+int FCConverter(void* ctx, OpLite* op, KernelBase* kernel) {
+  CHECK(ctx != nullptr);
+  CHECK(op != nullptr);
+  auto graph = static_cast<Graph*>(ctx);
+  auto op_info = op->op_info();
+  auto op_type = op_info->Type();
+  auto scope = op->scope();
+  VLOG(3) << "[NNA] Converting " + op_type + "...";
+
+  CHECK(op_info->HasAttr("enable_int8") &&
```

Comment:
expand to two CHECKs:

```
CHECK(op_info->HasAttr("enable_int8");
CHECK(op_info->GetAttr<bool>("enable_int8"));
```

So it shows exactly which condition is broken when failed.

Reply:
Done